doc/configuring.rst

```diff
@@ -224,7 +224,7 @@ artifacts
     Path to the file or directory to be copied to the working directory as an execution result.
     Can contain shell-style pattern matching (e.g. `"out/*.html"`), including recursive wildcards
     (e.g. `"out/**/index.html"`).
-    If not stated otherwise (see ``--no-archive`` `command-line parameter <args.html#Artifact\ collection>`__
+    If not stated otherwise (see ``--no-archive`` `command-line parameter <args.html#Artifact\\ collection>`__
```

Comment:
The added slash break the link, see it here: https://universum.readthedocs.io/en/type-annot/configuring.html